src/_includes/components/job-listing.njk

```diff
@@ -5,8 +5,8 @@
   </div>
   <header class="c-job-listing__header">
     <time>Posted on {{ params.date | readableDate }}</time>
-    <h2 class="c-job-listing__title"><a href="">{{ params.title }} at {{ params.company }}</a></h2>
-    <p>Type <strong>{{ params.type }}</strong> &bullet; Location <strong>{% if params.remote === true %}Remote{% else %}{{ params.location }}{% endif %}</strong></p>
+    <h2 class="c-job-listing__title"><a href="">{{ params.title }} at {{ params.company }}</a></h2>    
+    <p>Type <strong>{{ params.type }}</strong> &bullet; Location <strong>{{ params.location }}</strong>{% if params.remote === true %} Remote &bullet; <strong>Available</strong>{% endif %}</p>
```

Comment:
What do you think about wrapping the location in a conditional that checks to see if they have any office?

So if the offices param is set to false, we don't output the location?

`{% if params.offices === true %} &bullet; Location <strong>{{ params.location }}</strong>{% endif %}`

Reply:
I originally had that and removed it for some reason. 

If offices === false, what should it say? Remote Only, or still Remote Available?

Reply:
I like that. Lets move forward with that.